### ISSUE TITLE
Allow Multitoken Prediction in MLM Models

### ISSUE THREAD

**Issue by LazerLambda** [LazerLambda/Promptzl#49]
As in openprompt

- [ ] Verbalizer-init-fun for MLM
  - [ ] Option for multi-token
  - [ ] Options for how to include multi-token 
- [ ] Verbalizer-init-fun for CLM
- [ ] Extraction function for logits
